### PR TITLE
Mic 945

### PR DESCRIPTION
**Pull request by JonahKohen**
Changes made to lower level mcc linux driver software as part of MIC-945 PR.

### CODE REVIEW

File: USB/mcc-libusb/usb-2416.c [hunk 1]
```diff
@@ -251,7 +258,8 @@ int usbAIn_USB2416(libusb_device_handle *udev, uint8_t channel, uint8_t mode, ui
     return -1;
   }
 
-  libusb_control_transfer(udev, requesttype, AIN, wValue, wIndex, (unsigned char *) &data, sizeof(data), HS_DELAY);
+  if (libusb_control_transfer(udev, requesttype, AIN, wValue, wIndex, (unsigned char *) &data, sizeof(data), HS_DELAY) < 0)
```
Comment (murataksoy) [inline USB/mcc-libusb/usb-2416.c:261]: Above you returned whatever the return value was. Here, you just return -1. Is there a reason for this?

Comment (murataksoy) [reply USB/mcc-libusb/usb-2416.c:261]: Also below in `usbReadMemory_USB2416_Error_Return`, you just return the return value as well.

**Review by murataksoy — APPROVED** [review]
A small comment regarding the consistency of return values. Otherwise, feel free to merge.

File: USB/mcc-libusb/usb-2416.c (hunk 1 above)
Comment (JonahKohen) [reply USB/mcc-libusb/usb-2416.c:261]: Yes. This is the function you and I were talking about that returns an analog reading, where -1 can correspond, in theory, to a valid return value. I want to return as few values as possible that can correspond to an error code, to minimize the potential amount of false negative readings.

**Pull request merged**